### PR TITLE
Translated to Greek

### PR DESCRIPTION
**Pull request by sk8harddiefast**
Finished. Not 100% sure about Trigger meaning

        <source>Trigger Terminal</source>
        <translation>Εμφάνιση/Απόκρυψη Γραμμής Εντολών</translation>

### CODE REVIEW

**Comment by beanpole135** [comment]
"Trigger Terminal" would be comparable to saying "Toggle Terminal" or something similar. Basically, that button changes the terminal between visible/invisible depending on which state it is currently in.

**Comment by beanpole135** [comment]
If you go think the translation is fine right now - I can go ahead and merge it. Otherwise just let me know when you commit that change and I can merge it in after that.

**Comment by sk8harddiefast** [comment]
It's Ok,. Εμφάνιση/Απόκρυψη means Show / Hide. The exact word of Trigger is Πυροδότηση but we use it more on weapons or when we want to describe the beginning of a situation

File: src-qt5/desktop-utils/lumina-terminal/i18n/l-terminal_el.ts [hunk 1]
```diff
@@ -6,40 +6,40 @@
...
         <location filename="../TrayIcon.cpp" line="123"/>
         <source>Trigger Terminal</source>
-        <translation type="unfinished"></translation>
+        <translation>Εμφάνιση/Απόκρυψη Γραμμής Εντολών</translation>
     </message>
     <message>
         <location filename="../TrayIcon.cpp" line="125"/>
         <source>Top of Screen</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>Επάνω Μέρος της Οθόνης</translation>
```
Comment (beanpole135) [inline src-qt5/desktop-utils/lumina-terminal/i18n/l-terminal_el.ts]: Looks like a typo here:
The first </translation> tag in line 27 should not be there - as it should be properly closed at the end of the line instead

**Comment by beanpole135** [comment]
Ok, I only found that one error in your commit - if you fix that on your end, the pull request will be automatically updated and I can merge it in to the main source tree.

**Comment by beanpole135** [comment]
Merged!
Thanks for sending that in!

**Pull request merged**